src/main/java/org/commcare/formplayer/session/MenuSession.java

```diff
@@ -440,4 +471,7 @@ public SerializableMenuSession serialize() {
         return session;
     }
 
+    public ArrayList<PeristentCommand> getPersistentMenu() {
+        return persistentMenuHelper.getPersistentMenu();
```

Comment:
why is it that `persistentMenuHelper` doesn't to explicitly define a `getPersistentMenu` function?

Reply:
Kotlin auto adds the getters and setters for vars without us needing to define them